pkg/controller/cf_update_handlers.go

```diff
@@ -342,12 +342,9 @@ func (env *CfEnvironment) createAppServiceGraph(appId string, extIps []string) (
 		// 2. Service graph contract and external network
 		serviceObjs = append(serviceObjs,
 			apicExtNet(name, cont.config.AciVrfTenant,
-				cont.config.AciL3Out, extIps))
+				cont.config.AciL3Out, extIps, DefaultServiceExtNetShared))
```

Comment:
Minor comment: I think in a follow up patch we should ideally change the naming convention to be "ExtSubnet" instead of "ExtNet".